public/js/app.Ctrl.js

```diff
@@ -0,0 +1,201 @@
+App.controller('App.Ctrl', function($scope, $timeout, AppService) {
+    const colorCode = {1:"green", 2: "red", 3: "yellow", 4: "blue"};
+    const speed = 500; // in milliseconds
+    var on = false;
+    var lock = false;
+    var strict = false;
+    var colorSequence = [];
+    var count = 0;
+    var round = 1;
+    var turnOffTimer;
+    var recursiveTimer;
+    var timer;
+    var sound = {1:{}, 2:{}, 3: {}, 4: {}};
+    var buzzer;
+    var win;
+
```

Comment:
This isn't an init method if it depends on 'on === true'.  Maybe leverage an angular directive to help check if on is true before calling an init method if possible.